### ISSUE TITLE
Mounts

### ISSUE THREAD

**Issue by babosh-gh** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1097]
**Describe your suggestion in full detail below:**

Ola

Is it possible implement mounts?
Ive seen a mod for vanilla ck2, which implemented warhorses as artifacts.
Imagine the possibilities, tons of culture-specific mounts with different colors and armors

For instance, you could get a mount by purchasing one (intrigue menu for all characters) OR by giving the"keeper of horses" minor title holder character a special skill which allows their overlord to buy or freely get mounts from them

**Comment by MemeWizard** [comment]
Yes, it possible, but, implementing things like pets/mounts as artifacts is dubious for many reasons.

What about looting and equipping someone else's loyal dog?
What about their age? I don't think horses have indefinite lifespans and can lasts forever, like other artifacts.
If we solve this, what about gifting your rare, but old dying cat to other character to get a lot of positive opinion, because of rarity?
Etc, etc.

There a lot of things that needs to be reworked if someone wants to add a living creatures as artifacts, otherwise it will look extremely crutch-y.

**Comment by babosh-gh** [comment]
@MemeWizard you are right, implementing them as artifacts is clonky. How about implementing them as the ASOIAF mod implemented dragons?

**Comment by Gartarr** [comment]
@MemeWizard This could be done as trait or modifier. You know, like dogs and cats in vanilla. Add MTTH to remove trait/modifier and that's it. I know it's so 2000's but if it works it works.

**Comment by MemeWizard** [comment]
> How about implementing them as the ASOIAF mod implemented dragons?

Implementing them as characters require even more crutches. Like, I think I will not exaggerate if I say at least 100 times more.

But, yeah, they can be implemented as modifiers, like all other pets in vanilla/mod, or, after a bit of polishing, as artifacts.
